cfme/tests/configure/test_access_control.py

```diff
@@ -349,9 +353,12 @@ def test_rolename_duplicate_validation():
 
 
 @pytest.mark.tier(3)
+@pytest.mark.meta(blockers=[1351897])
 def test_delete_default_roles():
-    flash_msg = \
-        'Role "{}": Error during \'destroy\': Cannot delete record because of dependent miq_groups'
+    flash_msg = version.pick({'5.6': ("Role \"{}\": Error during delete: Cannot delete record " +
```

Comment:
```python
flash_msg = version.pick({
    '5.6': ("Role \"{}\": Error during delete: Cannot delete record "
        "because of dependent entitlements"),
    '5.5': ("Role \"{}\": Error during \'destroy\': Cannot delete record "
        "because of dependent miq_groups")})
```